apps/code-examples/src/app/code-examples/modals/confirm/confirm-demo.component.html

```diff
@@ -1,6 +1,6 @@
 <button
   aria-haspopup="dialog"
-  class="sky-btn sky-btn-default sky-margin-inline-sm"
+  class="sky-btn sky-btn-default sky-margin-inline-lg"
```

Comment:
Is this change correct? Elsewhere, we change `default` to `lg`, so it seems odd to change `sm` to `lg` unless `sm` was a bug in the first place.

Reply:
Good catch. I'm not sure how that one got changed!